packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -133,6 +134,25 @@ export function encodePubKey(value: string) {
   }
   return b58cencode(value.substring(2, 42), prefix.KT);
 }
+
+/**
+ *
+ * @description Base58 encode an address using predefined prefix (tz1, tz2, tz3). KT1 is not supported.
+ *
+ * @param value Address to base58 encode (tz1, tz2, tz3 or KT1)
+ */
+export function encodeAddress(value: string) {
```

Comment:
My thoughts on this function [here](https://github.com/ecadlabs/taquito/issues/2372#issuecomment-1877902966), do you mind adding the strip '0x' logic and disclaim it's limitation with originated address with annots?

Reply:
that's a good idea, I'll update

Reply:
thanks much for the update, prior your change the description marked kt1 is not supported which isn't true, am hoping we can removed change the description to  

 * @description Base58 encode an address using predefined prefix tz1, tz2, tz3 or KT1(without annotation). 
 * @param value Address to base58 encode tz1, tz2, tz3 or KT1, supports value with and without prefix `0x`

Reply:
updated